UI/Web/src/theme/themes/dark.scss

```diff
@@ -2,6 +2,16 @@
 
 :root, :root .default {
     /* Base colors */
+    --layer0: rgba(255,255,255,0);
```

Comment:
What are these layers? It's not clear from usage either. 

I'm not sure why we have a material file either?

Reply:
Changed the names to make it more clear and some documentation, but they refer to elevation layers.
https://m2.material.io/design/color/dark-theme.html#properties
![image](https://github.com/Kareadita/Kavita/assets/7907265/47fe102b-c552-4e36-84b0-3d84d89cceba)